crates/extension_host/src/wasm_host/wit/since_v0_3_0.rs

```diff
@@ -664,14 +664,21 @@ impl ExtensionImports for WasmState {
                             })
                             .cloned()
                             .unwrap_or_default();
-                        Ok(serde_json::to_string(&settings::ContextServerSettings {
-                            command: settings.command.map(|command| settings::CommandSettings {
-                                path: Some(command.path),
-                                arguments: Some(command.args),
-                                env: command.env.map(|env| env.into_iter().collect()),
-                            }),
-                            settings: settings.settings,
-                        })?)
+                        match settings {
+                            ServerConfig::Stdio { command, settings } => {
+                                Ok(serde_json::to_string(&settings::ContextServerSettings {
+                                    command: command.map(|command| settings::CommandSettings {
+                                        path: Some(command.path),
+                                        arguments: Some(command.args),
+                                        env: command.env.map(|env| env.into_iter().collect()),
+                                    }),
+                                    settings,
+                                })?)
+                            }
+                            ServerConfig::Sse { .. } => {
+                                bail!("SSE server configuration is not supported")
+                            }
+                        }
```

Comment:
For now, I didn't add support in the extension, because I didn't want to mess around with the API. It seems that `since_v0_2_0` is using `since_v0_3_0` (under the name `latest`). Also this depends on the general settings